app/views/shared/_money_input.html.slim

```diff
@@ -1,3 +1,7 @@
-= form.govuk_number_field field, width:,
-        label: { text: label_text },
-        prefix_text: BuildEstimatesHelper::POUND
+= form.govuk_text_field field,
+                        width:,
+                        label: { text: label_text },
+                        prefix_text: BuildEstimatesHelper::POUND,
+                        value: decimal_as_money_string(form, field),
+                        pattern: "[0-9]*",
+                        inputmode: "numeric"
```

Comment:
Having approved this, I just wondered if we should have had a test that demonstrates adding commas during the input of a mortgage (say). WDYT?

Reply:
Good idea, will add!

Reply:
Thank goodness you suggested this, because there was a bug that the test demonstrated!